### PR TITLE
feat[HACBS-806]:Add unit test for tekton functions

### PR DESCRIPTION
**Pull request by jinqi7**
Signed-off-by: Jing Qi <jinqi@redhat.com>

### CODE REVIEW

**Review by dirgim — APPROVED** [review]
LGTM

File: tekton/predicates_test.go [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+				Params: []tektonv1beta1.Param{
+					{
+						Name: "output-image",
+						Value: tektonv1beta1.ArrayOrString{
+							StringVal: "test-image",
+						},
+					},
+				},
+			},
+			Status: tektonv1beta1.PipelineRunStatus{
```
Comment (davidmogar) [inline tekton/predicates_test.go:46]: Maybe I'm wrong, but this should have no effect at all. You cannot set statuses this way.

Comment (dirgim) [reply tekton/predicates_test.go:46]: This got me thinking and I tried it out locally - the status does set in the `PipelineRun` object (hence why the unit tests are passing), but it doesn't actually get applied if we wanted to create that resource on the cluster.

Comment (davidmogar) [reply tekton/predicates_test.go:46]: My bad. You are absolutely right. Most of our tests create objects so I didn't realize you don't do it here.

**Review by Dannyb48 — APPROVED** [review]
LGTM

**Pull request merged**